main.yml

```diff
@@ -11,8 +11,8 @@
   - include: compose.yml
 
   - name: Wait for 10s to Get the IP
-    pause:
-      seconds: 10
+    wait_for:
```

Comment:
At the moment it is difficult to include a condition, but in the future this is very handy to use... as you can continue when a condition is met, like a host reachable, or port available, etc.